examples/data-transfer-mssql/main.py

```diff
@@ -70,13 +71,13 @@ def get_connection(db_host, db_name, db_username, db_password):
 def populate_data(RECORD_COUNT: int, db_cursor):
     ''' Generate user data. '''
     fake = Faker()
-    row = lambda n: (n + 1, repr(fake.name()), repr(fake.city()))
+    row: Callable[[int], Tuple[int, str, str]] = lambda n: (n + 1, repr(fake.name()), repr(fake.city()))
 
     for i in range(RECORD_COUNT):
         db_cursor.execute(SQL_INSERT_DATA, row(i))
 
 
-def extract_sql(file: str):
+def extract_sql(file):
```

Comment:
?

Reply:
I am not sure about the type here. 
I tried `str` but mypy suggested `str has no attribute read`